<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kalibr</class>
 <widget class="QDialog" name="Kalibr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калибровка осциллографа</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonAccept">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Принять</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pushButtonClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditZeroIb"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Нулевой уровень:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Сигнал тока фазы В&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Нулевой уровень:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="labelValueIb">
        <property name="text">
         <string>0 А</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonZeroApplyIb">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButtonZeroApplyUb">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_33">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Текущее значение:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Сигнал напряжения фазы В&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Задать значение:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Задать значение:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QLabel" name="labelValueUb">
        <property name="text">
         <string>0 В</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditZeroUb"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_36">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Текущее значение:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditValueUb"/>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButtonValueApplyUb">
        <property name="text">
         <string>Установить</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditValueIb"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonValueApplyIb">
        <property name="text">
         <string>Установить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButtonZeroApplyUc">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_34">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Текущее значение:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Сигнал напряжения фазы C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditZeroIc"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Нулевой уровень:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditZeroUc"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Задать значение:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>Задать значение:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Нулевой уровень:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonZeroApplyIc">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Сигнал тока фазы C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_37">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Текущее значение:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="labelValueIc">
        <property name="text">
         <string>0 А</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QLabel" name="labelValueUc">
        <property name="text">
         <string>0 В</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditValueIc"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonValueApplyIc">
        <property name="text">
         <string>Установить</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditValueUc"/>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButtonValueApplyUc">
        <property name="text">
         <string>Установить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="labelValueIa">
        <property name="text">
         <string>0 А</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Задать значение:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonZeroApplyIa">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButtonZeroApplyUa">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_31">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Текущее значение:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Нулевой уровень:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonValueApplyIa">
        <property name="text">
         <string>Установить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Нулевой уровень:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Задать значение:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditValueIa"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditZeroIa"/>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Сигнал напряжения фазы А&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditZeroUa"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Сигнал тока фазы А&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_35">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Текущее значение:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QLabel" name="labelValueUa">
        <property name="text">
         <string>0 В</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditValueUa"/>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButtonValueApplyUa">
        <property name="text">
         <string>Установить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditZeroIa</tabstop>
  <tabstop>pushButtonZeroApplyIa</tabstop>
  <tabstop>lineEditValueIa</tabstop>
  <tabstop>lineEditZeroUa</tabstop>
  <tabstop>pushButtonZeroApplyUa</tabstop>
  <tabstop>lineEditValueUa</tabstop>
  <tabstop>lineEditZeroIb</tabstop>
  <tabstop>pushButtonZeroApplyIb</tabstop>
  <tabstop>lineEditValueIb</tabstop>
  <tabstop>lineEditZeroUb</tabstop>
  <tabstop>pushButtonZeroApplyUb</tabstop>
  <tabstop>lineEditValueUb</tabstop>
  <tabstop>lineEditZeroIc</tabstop>
  <tabstop>pushButtonZeroApplyIc</tabstop>
  <tabstop>lineEditValueIc</tabstop>
  <tabstop>lineEditZeroUc</tabstop>
  <tabstop>pushButtonZeroApplyUc</tabstop>
  <tabstop>lineEditValueUc</tabstop>
  <tabstop>pushButtonAccept</tabstop>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>lineEdit_12</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
